### PR TITLE
updated routes endpoint data

### LINKED ISSUE #119: Adding endpoints info on the homepage.
**Describe the feature**
Adding following endpoints info on the homepage.
- [ ] /shows?genre=abc
- [ ] /shows?rating=def
- [ ] /shows?seasons=ghi
- [ ] /books?author=abc
- [ ]  /books?price=123
- [ ] /books?name=xyz

**Are you working on this feature? (Yes)**

### PR DESCRIPTION
**Pull request by ritik307**
# Related Issue

-   closes #119 

# Proposed Changes
- [x] /shows?genre=abc
- [x] /shows?rating=def
- [x] /shows?seasons=ghi
- [x] /books?author=abc
- [x]  /books?price=123
- [x] /books?name=xyz

# Checklist

-   [x] Tests
-   [x] Translations

### CODE REVIEW

**Comment by ritik307** [comment]
@ShauryaAg @paramsiddharth Plz review

**Comment by paramsiddharth** [comment]
@ritik307 Would you share a few screenshots of the updated homepage?

**Comment by ritik307** [comment]
@paramsiddharth Here you go!
**Shows**
image
image
image
**Books**
image
image
image

File: src/views/data.ts [hunk 1]
```diff
@@ -177,7 +177,8 @@ const data = [
     },
     {
         heading: "FoodPhotos",
-        desc: "The foodphotos route returns photos of food item with their source link,name and taste.",
+        desc:
+            "The foodphotos route returns photos of food item with their source link,name and taste.",
```
Comment (ShauryaAg) [inline src/views/data.ts:181]: why is this change needed?

File: src/views/data.ts [hunk 2]
```diff
@@ -219,7 +220,8 @@ const data = [
     },
     {
         heading: "Animal",
-        desc: "The animal route returns dummy data realted to animals with their type,breed and photos.",
+        desc:
+            "The animal route returns dummy data realted to animals with their type,breed and photos.",
```
Comment (ShauryaAg) [inline src/views/data.ts:224]: why is this change needed?

File: src/views/data.ts [hunk 3]
```diff
@@ -646,7 +648,7 @@ const data = [
             {
                 route: "GET /songs?genre=pop&language=punjabi",
                 content:
-          "Return dummy songs data after chaining genre and language query parameter.",
+                    "Return dummy songs data after chaining genre and language query parameter.",
```
Comment (ShauryaAg) [inline src/views/data.ts:651]: correct this, please.

File: src/views/data.ts [hunk 4]
```diff
@@ -731,7 +733,8 @@ const data = [
         routes: [
             {
                 route: "GET /shows/",
-                content: "Return all the available shows with title,genre,season,episodes,rating,description.",
+                content:
+                    "Return all the available shows with title,genre,season,episodes,rating,description.",
```
Comment (ShauryaAg) [inline src/views/data.ts:737]: Same for this.

**Review by ShauryaAg — CHANGES_REQUESTED** [review]
Please undo all the unnecessary changes.

**Comment by ritik307** [comment]
@ShauryaAg Done! plz review

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

**Comment by ritik307** [comment]
@ShauryaAg Done!

**Review by ShauryaAg — APPROVED** [review]
